### ISSUE TITLE
Add the ability to send the bots to a desired location

### ISSUE THREAD

**Issue by Auditormadness9** [nullworks/cathook#1420]
**Quick Description**

I would like to suggest the ability to send bots to any location of choosing on the map, via pointing at a place in the game with mouse for example (crosshair).

**The Issue**

I know making them followbot and leading them is a good alternative, but I keep getting slaughtered on my way into a suicide mission in an enemy sentry nest unless I am ubered, and once I am dead, the bots will stand still, in most occasions in a very uncomfortable position, where they'll get absolutely obliterated. I would like for a "send to location" feature to be added, so that I could send my heavybots (perhaps sniperbots too) into the nest and proceed after them once they clear, or assist them in it (they could also just be decoys, would be cool for scoutbots, as sentry knockbacks are literal hell, especially when the entire team is conscious of the bots and went full engineer gaming with 7 sentries). Maybe perhaps make the heavies go there and have some pocket medics of theirs follow them, like a tank march.

**My Suggestion for Developers**

A good way to implement this would be a simple cat command, that we all will obviously bind to a key, unless we ran out of unbinded keys.

**Additional Context**

Time to end the days of getting flexed on by 400 legit engineers and getting told how shit our bots are. That's our only weak point... without engineer gaming they are nothing, even with 100 heavies with 50 vaccinator medics.

**Comment by iraizo** [comment]
This sounds like a more precise version of https://github.com/nullworks/cathook/issues/1093, i dont think the crosshair location would be needed like that but a easier implementation would be the "Go" command for the bots to move to your current position and stay there

**Comment by Auditormadness9** [comment]
>i dont think the crosshair location would be needed like that but a easier implementation would be the "Go" command for the bots to move to your current position and stay there

Not really, we need exactly via crosshair. As I explained, the issue is entering a certain room or area where there are many sentries flanking you. There's no point to make the bot go to your location via the go command, it's pretty much the same as followbot except it's not constant, you'll get obliterated even more before you play out your "pivot" role of bringing them to the area.

**Comment by iraizo** [comment]
I guess raytrace until it hits a solid and use that as a location

**Comment by EngineerGaming69** [comment]
i think lithium tried this before and it never became useful or a thing

**Comment by iraizo** [comment]
I also find this very useless.

**Comment by EngineerGaming69** [comment]
> I also find this very useless.

the bots already have big brains when fighting sentrys if u have target sentrys on

**Comment by iraizo** [comment]
> > I also find this very useless.
> 
> the bots already have big brains when fighting sentrys if u have target sentrys on

Exactly, i was very impressed on how they handles sentries, they were very good at it even when they sometimes got some hickups with the sentry range

**Comment by Auditormadness9** [comment]
> the bots already have big brains when fighting sentrys if u have target sentrys on

They do, but don't you think it's a little inefficient peaking through corner trying to shoot a sentry when an entgie spams the wrench on it from behind with a dispenser behind him, and 6 more fucking engies with their own sentries there? Much more inefficient than rushing into the middle of the area, getting ubered and minigunning everyone, then you go in. (Also I don't want to followbot them into it even ubered, cuz sentry knockback is horrible, but it can't handle 6 ubered heavies rushing in at once)
